### PR TITLE
Ejudge Cookie Auth and CHU2024 configs

### PR DESCRIPTION
**Pull request by ArsNotFound**

### CODE REVIEW

File: src/cds/plugins/ejudge/src/main/kotlin/org/icpclive/cds/plugins/ejudge/EjudgeDataSource.kt [hunk 1]
```diff
@@ -25,6 +25,10 @@ public sealed interface EjudgeSettings : CDSSettings {
         get() = TimeZone.of("Europe/Moscow")
     public val problemScoreLimit: Map<String, Double>
         get() = emptyMap()
+    public val authCookieName: String
+        get() = ""
+    public val authCookieValue: String
+        get() = ""
```
Comment (Mond1c) [inline src/cds/plugins/ejudge/src/main/kotlin/org/icpclive/cds/plugins/ejudge/EjudgeDataSource.kt:31]: Maybe null instead of empty string?

**Review by Mond1c — COMMENTED** [review]
OK, but run ./gradlew apiDump and ./gradlew :schema-generator:gen, because now the tests fail. Then push the changes.

File: config/chu/2024/d1/settings.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "ejudge",
+  "url": "https://timus.online/admin/spacelog.aspx/log-1605.xml?space=1605&action=getejudgelog",
```
Comment (Mond1c) [inline config/chu/2024/d1/settings.json:3]: "url": "log-1605.xml"?

File: config/chu/2024/d2/settings.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "ejudge",
+  "url": "https://timus.online/admin/spacelog.aspx/log-1606.xml?space=1606&action=getejudgelog",
```
Comment (Mond1c) [inline config/chu/2024/d2/settings.json:3]: "url": "log-1606.xml"?

**Comment by ArsNotFound** [comment]
> OK, but run ./gradlew apiDump and ./gradlew :schema-generator:gen, because now the tests fail. Then push the changes.

Fixed in latest commits

**Review by Mond1c — APPROVED** [review]
OK

**Pull request merged**